docs/mockups/styles.css

```
@@ -0,0 +1,1191 @@
+  --red: #c83232;
+  --red-soft: #fff0ee;
+  --amber: #a15b00;
+  --amber-soft: #fff4df;
+  --green: #25734b;
+  --green-soft: #eaf7ef;
+  --blue: #2164a8;
+  --blue-soft: #eaf2fb;
+  --focus: #7654e6;
+  --r1: 3px;
+  --r2: 6px;
+  --sans: Inter, ui-sans-serif, -apple-system, BlinkMacSystemFont, "Segoe UI", "Noto Sans JP", sans-serif;
```

Stylelint (17.14.0)

[error] Expected "Inter" to be "inter" (value-keyword-case)

(value-keyword-case)